packages/react-native-renderer/src/ReactFabricHostConfig.js

```diff
@@ -125,28 +125,32 @@ class ReactFabricHostComponent {
   }
 
   blur() {
-    TextInputState.blurTextInput(this);
+    if (this._internalInstanceHandle != null) {
```

Comment:
Are you concerned at all about the silent failure added with the null checks in this PR?

If these checks ever fail, doesn't that indicate something is wrong? Should we invariant or error?

Reply:
I just wanted to align behavior -- the current behavior for other methods is to noop when unmounted. I don't know what `TextInputState` does but since focusing/bluring unmounted doesn't make sense (even if the user asks for it), seems ok to just ignore same as we ignore measure calls.